plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -312,4 +314,42 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
       deactivateLocationComponent()
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Create a [LocationPuck2D] instance with an arrow bearing image
+ */
+fun LocationComponentPlugin.createDefault2DPuckWithBearing(context: Context) = LocationPuck2D(
```

Comment:
Shall we move these to LocationComponentExt.kt ? so these will be visible under `LocationComponentUtils` for java users.

Reply:
That's right.